Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/openshift/origin-must-gather:4.2.0 as builder
+FROM quay.io/openshift/origin-must-gather:4.3 as builder
```

Comment:
I see `4.3.0` as well. Can you confirm which version would be better to use?

Reply:
As of right now, 4.3.0 is the same as 4.3. I don't know if there are any future releases planned.

Reply:
Please consult this with Liora. She should know which one to use.